packages/patternlab/styleguide/source/_patterns/02-molecules/expandable-section.twig

```diff
@@ -5,10 +5,20 @@
   <ul class="ma__toc--hierarchy__accordion-content js-accordion-content">
     {% block expandableSectionLinks %}
       {% for link in expandableSection.linkItems %}
-        <li>
-          {% if link.icon %}
+        <li{{ (link.subItems|length > 0) ? ' class="ma__toc--hierarchy__accordion-content__subitems-container"' : '' }}>
+          {% if link.subItems|length > 0 %}
+
+            {% set sidebarHeading = { "title": link.title} %}
```

Comment:
Please make the sidebar heading level passable, and use comp heading with `sidebar: true` instead

Reply:
done.